### DISCUSSION TITLE
Could we use GitHub discussions instead of Slack?

### DISCUSSION THREAD

**Discussion by AlexTelon** [AlexTelon/FlexiCharge-Org#3] [General]
possible pros:
 * GitHub discussions surely wont delete all data older than 90 days like slack
 * Probably easier to integrate to the rest of the GitHub Ecosystem

Cons:
 * Sometimes it is nice to have non-public discussions. Either we need to set this up centrally or let students use discord themselves if they feel the need for non-public discussions?

**Comment by FlygandeScotty** [comment]
Yes, always preferable to hold discussions within the community / channel.  I need to read up more on this topic - as it pertains to this new feature in GitHub.  However, students will no doubt use whatever channel is preferable to them - Slack or Discord.  If I recall Slack was the choice for the University?
